src/components/CompanyHome/CompanyHome.tsx

```diff
@@ -0,0 +1,17 @@
+import { Title } from "@deskpro/app-sdk";
+import { Container } from "@/components/common";
+import type { FC } from "react";
+
+type Props = {
+  // todo: will be implemented in next PR
```

Comment:
Use `@todo: ` as that's easier to find in the codebase.

Reply:
fixed